src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -1199,4 +1204,45 @@
      */
     GeneratorType registerGeneratorType(String name, Callable<WorldGenerator> generator, DataContainer settings);
 
+    /*
+     * Gets a complete list of {@link MapColor}s including the non-base colors.
+     *
+     * @return The list of all {@link MapColor}s available to be used
+     */
+    List<MapColor> getCompleteMapColors();
```

Comment:
Why can't this simply be `getMapColors()`? Unless you were planning on adding another method with only the base colours.